package.json

```
@@ -4,7 +4,7 @@
   "description": "Atom API autocompletions",
```

```suggestion
  "description": "Pulsar API autocompletions",
```

Again might as well be thorough on these ones

---

I feel like this could also be `API autocompletions` but I will do this change for now.